Translate howto/logging

josix opened the pull request.

josix committed:
Translated using Weblate (argparse (generated) (argparse))

Currently translated at 69.0% (58 of 84 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://localhost/projects/python-doc-zhtw/howto/argparse/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 11.2% (25 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 16.6% (37 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 16.6% (37 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 16.6% (37 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 16.6% (37 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

milanochuang committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 16.6% (37 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 16.6% (37 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

milanochuang committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 38.7% (86 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix commented on howto/argparse.po:
```diff
@@ -13,16 +13,16 @@ msgstr ""
-"Language: zh_TW\n"
+"PO-Revision-Date: 2023-02-26 20:52+0000\n"
+"Last-Translator: Weblate Admin <josixwang@gmail.com>\n"
+"Language-Team: argparse (generated) <http://localhost/projects/"
+"python-doc-zhtw/howto/argparse/>\n"
+"Language: argparse\n"
```
**issue:** need to correct these two fields into original ones

josix commented on howto/logging.po:
```diff
@@ -9,15 +9,16 @@ msgstr ""
+"PO-Revision-Date: 2023-03-10 16:12+0000\n"
+"Last-Translator: jouan <jouanofficial@gmail.com>\n"
+"Language-Team: logging (generated) <http://weblate.andyjjrt.cc/projects/"
+"python-doc-zhtw/howto/logging/>\n"
+"Language: logging\n"
```
**issue:** need to correct these two fields into original ones

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 50.0% (111 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 50.0% (111 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
fix(typo): Fix typo (#1)

Co-authored-by: Josix <josixwang@gmail.com>

mattwang44 commented on howto/argparse.po:
```diff
@@ -259,7 +259,7 @@ msgstr "介紹選項參數"
 "So far we have been playing with positional arguments. Let us have a look on "
 "how to add optional ones::"
-msgstr ""
+msgstr "到目前為止，我們一直在玩位置參數。 讓我們看看如何添加可選的::"
```
```suggestion
msgstr "到目前為止，我們一直在使用位置引數。讓我們看看如何添加可選引數::"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -61,19 +62,22 @@ msgid ""
 "func:`critical`. To determine when to use logging, see the table below, "
 "which states, for each of a set of common tasks, the best tool to use for it."
 msgstr ""
+"紀錄（Logging）提供方便使用的一組簡單紀錄函式。這些函式包括：:func:`debug`、:"
```
logging 相關翻譯還很少，logging 一詞目前多翻為「日誌」，但可以討論一下

```suggestion
"日誌（Logging）提供方便使用的一組簡單紀錄函式。這些函式包括：:func:`debug`、:"
```

josix replied:
我覺得可以先用日誌 (noun) ，logging 作為動作的話翻譯成記錄，當初翻的時候還沒統一，chatgpt 也沒有統一後續會再檢查一下

mattwang44 replied:
Ok

mattwang44 commented on howto/logging.po:
```diff
@@ -83,64 +87,70 @@ msgstr ":func:`print`"
 ":func:`logging.info` (or :func:`logging.debug` for very detailed output for "
 "diagnostic purposes)"
-msgstr ""
+msgstr ":func:`logging.info` （用於一般輸出）（或者 :func:`logging."
+"debug` （用於診斷目的的非常詳細的輸出））"
```
```suggestion
msgstr ":func:`logging.info` （或者在需要診斷用的詳細輸出時使用 :func:`logging."
"debug` ）"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -83,64 +87,70 @@ msgstr ":func:`print`"
 ":func:`warnings.warn` in library code if the issue is avoidable and the "
 "client application should be modified to eliminate the warning"
-msgstr ""
+msgstr "如果問題是可以避免的，那麼客戶端應該修改以消除警告，請在庫代碼中使用 "
+":func:`warnings.warn` 警告"
```
```suggestion
msgstr "如果問題是可以避免的，請在函式庫程式中使用 :func:`warnings.warn` "
"警告，並應修改客戶端應用程式來消除警告"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -83,64 +87,70 @@ msgstr ":func:`print`"
 #: ../../howto/logging.rst:52
 msgid "Raise an exception"
-msgstr ""
+msgstr "引發異常"
```
```suggestion
msgstr "引發例外"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -83,64 +87,70 @@ msgstr ":func:`print`"
 "Report suppression of an error without raising an exception (e.g. error "
 "handler in a long-running server process)"
-msgstr ""
+msgstr "報告錯誤但不引發異常（如長時間執行的伺服器進程中的錯誤處理程序）"
```
```suggestion
msgstr "報告錯誤的抑制 (suppression) 但不引發例外（如長時間執行的伺服器行程中的錯誤處理程序）"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -83,64 +87,70 @@ msgstr ":func:`print`"
 ":func:`logging.error`, :func:`logging.exception` or :func:`logging.critical` "
 "as appropriate for the specific error and application domain"
 msgstr ""
+"根據錯誤及應用領域的特定需要，使用 :func:`logging.error`、:func:`logging."
```
```suggestion
"根據特定錯誤及應用領域的需求來選擇使用 :func:`logging.error`、:func:`logging."
```

mattwang44 commented on howto/logging.po:
```diff
@@ -83,64 +87,70 @@ msgstr ":func:`print`"
 "they are used to track. The standard levels and their applicability are "
 "described below (in increasing order of severity):"
-msgstr ""
+msgstr "日誌紀錄函數的命名方式是根據它們所用於追蹤的事件等級或嚴重程度。以下是標準等"
```
- logging 要統一用詞
- function 統一翻譯為「函式」

```suggestion
msgstr "日誌函數的命名方式是根據它們所用於追蹤的事件等級或嚴重程度。以下是標準等"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -188,21 +199,23 @@ msgstr "``CRITICAL``"
 "and above will be tracked, unless the logging package is configured to do "
 "otherwise."
-msgstr ""
+msgstr "預設的級別是「警告」(``WARNING``)，這意味著除非配置記錄器套件進行其他設定，否"
```
```suggestion
msgstr "預設的級別是「警告」(``WARNING``)，這意味著除非為日誌套件配置其他設定，否"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -188,21 +199,23 @@ msgstr "``CRITICAL``"
 "otherwise."
-msgstr ""
+msgstr "預設的級別是「警告」(``WARNING``)，這意味著除非配置記錄器套件進行其他設定，否"
+"則隻會追蹤這個級別及以上的事件。"
```
```suggestion
"則只會追蹤這個級別以上的事件。"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -188,21 +199,23 @@ msgstr "``CRITICAL``"
 "way is to write them to a disk file."
-msgstr ""
+msgstr "被追蹤的事件可以採取不同的處理方式。處理追蹤事件最簡單的方法是將它們輸出到控"
+"制台。另一個常見的方法是將它們寫入磁盤文件。"
```
```suggestion
"制台。另一個常見的方法是將它們寫入磁碟文件中。"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -228,10 +244,14 @@ msgid ""
 #: ../../howto/logging.rst:124
 msgid "Logging to a file"
-msgstr ""
+msgstr "將記錄寫入檔案"
```
```suggestion
msgstr "將日誌寫入檔案"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -228,10 +244,14 @@ msgid ""
 "explained later. The actual output can be formatted quite flexibly if you "
 "need that; formatting options will also be explained later."
 msgstr ""
+"這段訊息顯示在控制台上。由於預設級別為 WARNING，所以 INFO 訊息並未出現。"
```
```suggestion
"這段訊息顯示在控制台上。由於預設級別為 ``WARNING``，所以 ``INFO`` 訊息並未出現。"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -228,10 +244,14 @@ msgid ""
 "need that; formatting options will also be explained later."
 msgstr ""
+"這段訊息顯示在控制台上。由於預設級別為 WARNING，所以 INFO 訊息並未出現。"
+"印出的訊息包含級別以及在 logging 呼叫中提供的事件說明，例如 'Watch out!'。"
```
```suggestion
"印出的訊息包含級別以及在日誌呼叫中提供的事件說明，例如 'Watch out!'。"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -228,10 +244,14 @@ msgid ""
 msgstr ""
+"這段訊息顯示在控制台上。由於預設級別為 WARNING，所以 INFO 訊息並未出現。"
+"印出的訊息包含級別以及在 logging 呼叫中提供的事件說明，例如 'Watch out!'。"
+"目前不必擔心 'root' 部分，稍後將會說明。如果您需要的話，可以彈性地格式化實際"
```
```suggestion
"目前不必擔心 'root' 部分，稍後將會說明。如果有需要的話，可以彈性地格式化實際"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -240,6 +260,10 @@ msgid ""
 "Python interpreter, and don't just continue from the session described "
 "above::"
 msgstr ""
+"一個非常普遍的情況是將記錄事件記錄在文件中，因此讓我們接下來來看看這個情況。"
```
```suggestion
"一個非常普遍的情況是將日誌事件記錄在文件中，因此讓我們接著看看這個情況。"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -240,6 +260,10 @@ msgid ""
 "above::"
 msgstr ""
+"一個非常普遍的情況是將記錄事件記錄在文件中，因此讓我們接下來來看看這個情況。"
+"請務必在新啟動的 Python 解譯器中嘗試以下操作，而不是繼續以上描述的會話：\n"
```
```suggestion
"請務必在新啟動的 Python 直譯器中嘗試以下操作，而不是繼續以上描述的會話：\n"
```

mattwang44 commented on howto/logging.po:
```diff
@@ -249,30 +273,35 @@ msgid ""
 "passed, which determines how encoding errors are handled. For available "
 "values and the default, see the documentation for :func:`open`."
 msgstr ""
+"新增了 *encoding* 參數。在早期的 Python 版本中，如果未指定，將使用 "
```
```suggestion
"新增了 *encoding* 引數。在早期的 Python 版本中，如果未指定，將使用 "
```

mattwang44 commented on howto/logging.po:
```diff
@@ -249,30 +273,35 @@ msgid ""
 msgstr ""
+"新增了 *encoding* 參數。在早期的 Python 版本中，如果未指定，將使用 "
+":func:`open` 的預設編碼。儘管在上面的範例中沒有顯示，現在也可以傳遞 *errors* "
+"參數，以決定如何處理編碼錯誤，詳情請參閱 :func:`open` 的文件。"
```
```suggestion
"引數，以決定如何處理編碼錯誤。可用的值與其預設值，請參閱 :func:`open` 的文件。"
```

mattwang44 reviewed (changes requested):
review round 1

josix committed:
Update howto/argparse.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

josix committed:
Update howto/logging.po

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

Commit:
Translated using Weblate (enum (generated) (enum))

Currently translated at 0.4% (1 of 224 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/enum/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 50.0% (111 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 50.0% (111 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 50.4% (112 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 52.2% (116 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 52.2% (116 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 52.2% (116 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 52.2% (116 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 59.0% (131 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 59.0% (131 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 59.0% (131 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 59.0% (131 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 59.0% (131 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 62.1% (138 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 62.1% (138 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 62.1% (138 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 63.0% (140 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 63.0% (140 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 63.0% (140 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 63.0% (140 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 77.4% (172 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 77.4% (172 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 77.4% (172 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

mingray0121 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 77.4% (172 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 77.4% (172 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 79.2% (176 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 79.2% (176 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 80.6% (179 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 80.6% (179 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (logging (generated) (logging))

Currently translated at 98.6% (219 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

andyjjrt committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 98.6% (219 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (regex (generated) (regex))

Currently translated at 20.9% (60 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/

eastrong23 committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 98.6% (219 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

josix committed:
Translated using Weblate (logging (generated) (logging))

Currently translated at 98.6% (219 of 222 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/logging/

Commit:
Translated using Weblate (functional (generated) (functional))

Currently translated at 3.3% (7 of 206 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/functional/

mingray0121 committed:
Translated using Weblate (regex (generated) (regex))

Currently translated at 40.5% (116 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/

eastrong23 committed:
Translated using Weblate (regex (generated) (regex))

Currently translated at 40.5% (116 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/

Commit:
Translated using Weblate (regex (generated) (regex))

Currently translated at 40.5% (116 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/

josix committed:
Translated using Weblate (regex (generated) (regex))

Currently translated at 40.5% (116 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/

mingray0121 committed:
Translated using Weblate (regex (generated) (regex))

Currently translated at 46.1% (132 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/

Commit:
Translated using Weblate (regex (generated) (regex))

Currently translated at 46.1% (132 of 286 strings)

Translation: python-doc-zhtw/howto
Translate-URL: http://weblate.andyjjrt.cc/projects/python-doc-zhtw/howto/regex/
